### ISSUE TITLE
bug: If "See Also" is first, initial description is omitted

### ISSUE THREAD

**Issue by NickCrews** [machow/quartodoc#283]
with a docstring of

```
"""
one liner, this isn't shown

longer description, this isn't shown

See Also
--------
[related function, this is shown](#related_function)
""""
```

only the See Also section is rendered. The one liner and longer description is omitted. To get them to appear, I have to put the See Also section AFTER the other sections eg Returns:

```
"""
one liner, this is shown

longer description, this is shown

Returns
-------
a value, this is shown

See Also
--------
[related function, this is shown](#related_function)
""""
```

**Comment by machow** [comment]
Shoot--I think this might be a subtle issue upstream in griffe. Basically, for various reasons, it doesn't attempt to parse the See Also section. When See Also follows another section it does parse, this works out okay, and quartodoc is able to catch a represent it downstream.

However, it appears that when a section griffe doesn't parse gets included first, it's lumped in with everything before it. So this also does not represent the `Zzzzz` section as its own thing, but lumps it with everything above it.

```python
def f():
    """
    one liner, this isn't shown

    longer description, this isn't shown

    Zzzzz
    --------
    [related function, this is shown](#related_function)
    """

```

I'll open an issue upstream on griffe

**Comment by machow** [comment]
opened here: https://github.com/mkdocstrings/griffe/issues/214. Going to close here, and will sync upstream!

**Comment by machow** [comment]
I just realized we can fix this in some of the custom transforming we do to represent the SeeAlso section, and opened a PR (but will also be useful to contribute some chopping up section logic upstream!)

**Comment by machow** [comment]
Should be fixed in latest v0.6.4. Here is the docstring we test with

```
A short description

See Also
---
```
